### PR TITLE
[REQUEST]: Add information for installing TgCli with Guix.

### PR DESCRIPTION
**Pull request by raghavgururajan**
@erayerdin 

I recently [packaged](https://git.savannah.gnu.org/cgit/guix.git/commit/?id=49d8f954e631abaec08fc58f58cbfd13c3aabc41) TgCli into Guix.

Could you please add information for installing TgCli with Guix, in README file?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +52,13 @@ Also available as a package in [AUR](https://aur.archlinux.org/packages/tgcli/).
 # assuming you use yay
 yay -S tgcli
 ```
+### Install with [Guix](https://guix.gnu.org)
+
+TgCli is [available](https://guix.gnu.org/en/packages/tgcli-0.3.1/) in official channel of Guix.
```
Comment (erayerdin) [inline README.md:57]: Just a slight thing. I think it should be "tgcli", not "TgCli".

Comment (erayerdin) [reply README.md:57]:
```suggestion
tgcli is [available](https://guix.gnu.org/en/packages/tgcli-0.3.1/) in official channel of Guix.
```

**Review by erayerdin — CHANGES_REQUESTED** [review]
Thank you for contribution. There's a *slight* thing. I refer to the project as "tgcli" in many places, not "TgCli".

**Comment by erayerdin** [comment]
Ok, seems like I can also change it. I fixed the typo and will merge asap.

**Comment by raghavgururajan** [comment]
@erayerdin 

Thanks for the merge. :)

**Pull request merged**